openshift_cli_installer/libs/clusters/ipi_cluster.py

```diff
@@ -111,6 +116,15 @@ def _create_install_config_file(self):
         with open(os.path.join(self.cluster_info["cluster-dir"], "install-config.yaml"), "w") as fd:
             fd.write(yaml.dump(cluster_install_config))
 
+    def _set_docker_config_file(self):
+        ipi_docker_config_dir = os.path.dirname(self.docker_config_file)
+        if not ipi_docker_config_dir:
```

Comment:
```suggestion
        if not ipi_docker_config_dir := os.path.dirname(self.docker_config_file):
```

Reply:
thanks for suggestion @omrirh, the code is updated to use unified pull secrets now, to support latest oc cli version.